doc/source/openfisca-run-test.rst

```diff
@@ -0,0 +1,230 @@
+==================
+openfisca-run-test
+==================
+
+.. argparse::
+   :module: openfisca_core.scripts.run_test
+   :func: build_parser
+   :prog: openfisca-run-test
+
+Examples
+--------
+
+Let's assume that in the country package ``openfisca_france``, ``net_salary`` is always 80% of ``gross_salary``.
+
+Basic use
+^^^^^^^^^
+
+**test.yaml:**
+
+.. code-block:: yaml
+
+    - name: "Basic test"
+      period: 2015
+      input_variables:
+        gross_salary: 2000
+      output_variables:
+        net_salary: 2000 * 0.8
+
+**Command line:**
+
+.. code-block:: shell
+
+  openfisca-run-test -c openfisca_france test.yaml
+  # Success
+
+  openfisca-run-test test.yaml
+  # Success: the country package is automatically detected.
+  # May fail if several country packages are installed in your environment.
+  # In that case, specify which package to use with the --country_package option
+
+
+Force option
+^^^^^^^^^^^
+
+
+**test_2.yaml:**
+
+.. code-block:: yaml
+
+    - name: "Basic test"
+      period: 2015
+      input_variables:
+        gross_salary: 1000
+      output_variables:
+        net_salary: 800
+
+    - name: "Failing test"
+      ignore: True
+      period: 2015
+      input_variables:
+        gross_salary: 1000
+      output_variables:
+        net_salary: 0
+
+
+**Command line:**
+
+.. code-block:: shell
+
+  openfisca-run-test test_2.yaml
+  # Success: the second test is ignored
+
+  openfisca-run-test -f test_2.yaml
+  # Failure: the second test is executed and does not pass
+
+
+Error margin
+^^^^^^^^^^^^
+
+**test_3.yaml:**
+
+.. code-block:: yaml
+
+    - name: "Test defining its own relative error margin"
+      period: 2015
+      relative_error_margin: 0.05
+      input_variables:
+        gross_salary: 1000
+      output_variables:
+        net_salary: 780 # the right value is 800
+
+    - name: "Test defining its own absolute error margin"
+      absolute_error_margin: 10
+      period: 2015
+      input_variables:
+        gross_salary: 1000
+      output_variables:
+        net_salary: 790 # the right value is 800
+
+    - name: "Test not defining any error margin"
+      period: 2015
+      input_variables:
+        gross_salary: 1000
+      output_variables:
+        net_salary: 795 # the right value is 800
+
+
+**Command line:**
+
+.. code-block:: shell
+
+  openfisca-run-test test_3.yaml
+  # Failure: the third test does not pass, as its error margin is by default 0
+
+  openfisca-run-test -m 0.01 test_3.yaml
+  # Success:
+  # The first test passes with its own 0.05 relative error margin
+  # The second test passes with its own 10 absolute error margin
+  # The third test passes with the default 0.01 relative error margin
```

Comment:
This could benefit from a short theoretical explanation of what is an error margin and why it is useful… Might not be part of this PR though.